sdk/core/azure-core-tracing-opencensus/README.md

```diff
@@ -19,7 +19,7 @@ documentation][api_documentation] | [Samples][samples]
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-core-tracing-opencensus</artifactId>
```

Comment:
@hemantkr Do we usually update the version of the parent/pom too before individual package releases?

Reply:
no unless they are changed.